script/optimization_concepts/conic_programming.jl

```diff
@@ -132,12 +132,12 @@ optimize!(model)
 
 #+
 
-e1 = [1, zeros(10)...]
+e1 = [1.0, zeros(10)...]

```

Comment:
The int -> float conversions are not necessary in that case, since the zeros(10) will promote everything. Same below since these are constraint specifications ints are converted

Reply:
You are right, indeed. I will update the PR accordingly.